Chandan/chandan_practice.c

```diff
@@ -1,106 +1,199 @@
-#include <stdio.h>
-#include <time.h>
-#include <math.h>
-
-#define SECONDS_PER_MINUTE 60
-#define MINUTES_PER_HOUR 60
-#define HOURS_PER_DAY 24
-#define PROMPT_STRING "Type any key to show time, q to show time then quit: "
-
-#define TIME_TYPE unsigned long 
-
-char getChar() ;
-void updateTimes() ;
-void initTimes() ;
-void printPrompt() ;
-void printTime (TIME_TYPE timeInit) ;
-
-TIME_TYPE start, end;
+#include<stdio.h>
+#include<math.h>
+#include<stdlib.h>
+#include<string.h>
+#define MAX 50
+int top = -1, front = 0;
+int stack[MAX];
+void push(char);
+void pop();
+void addn(float,float, float);
 
 int hello_c()
 {
-        char command = 0;
-        initTimes();
-        delay();
-        updateTimes();
-        printTime(end-start);
-        return 0;
-}
+   float a = 14.5;
+   float b = 16;
+   float c = 17.5;
+   float *p, *q, *r;  
+   p = (int*)malloc(sizeof(int));
+   q = (int*)malloc(sizeof(int));
+   r = (int*)malloc(sizeof(int));
+   p = &a; 
+   q = &b;
+   r = &c;
+   int x = 3;
+   int y = 6;
+   int z = 5;
+   int i, HCF, max, step, LCM;
+   int j;
+   char s[MAX] = "radar", g;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'g' can be reduced.](https://app.codacy.com/manual/e-mock/CProgramming-Team6/pullRequest?prid=5597669)